packages/dds/tree/src/feature-libraries/object-forest/objectForest.ts

```diff
@@ -438,16 +448,7 @@ class Cursor extends SynchronousCursor implements ITreeSubscriptionCursor {
 
     fork(observer?: ObservingDependent): ITreeSubscriptionCursor {
```

Comment:
Do you know if we need to use the @inheritDoc tag for this to generate the API docs correctly or does it automatically inherit from the interface it implements?

Reply:
The main thing I care about is hover text in VS code, which works fine without inheritdoc. I'm not sure about generated API docs though

Reply:
That's fair. We definitely have a lot of cases where we don't use inheritDoc anyways so I'll look into it as part of my linking investigation.